src/tests/multihost/ad/conftest.py

```diff
@@ -111,6 +112,68 @@ def remove_cert():
         session_multihost.client[0].run_command(remove_client_ad_cacert)
     request.addfinalizer(remove_cert)
 
+@pytest.fixture(scope="function")
+def range_retr_mods(session_multihost, fetch_ca_cert, request):
+    """ AD-ldap modifications """
+    ad = ADOperations(session_multihost.ad[0])
+    ad_ldap = ad.ad_conn()
+    ad_password = session_multihost.ad[0].ssh_password
+    # set MaxValRange=$AD_MAX_RANGE on AD server and everything else to default
+    basedn = session_multihost.ad[0].domain_basedn_entry
+    DEF_QUERY= f'cn=Default Query Policy,cn=Query-Policies,cn=Directory Service,cn=Windows NT,cn=Services,cn=Configuration,{basedn}'
+    cmd = f'ldapsearch -x -h {session_multihost.ad[0].ip} -D "cn=Administrator,cn=Users,{basedn}" -w {ad_password} -b "{DEF_QUERY}" lDAPAdminLimits |egrep MaxValRange|cut -d "=" -f2'
+    value = session_multihost.client[0].run_command(cmd)
+    orig_val = value.stdout_text.rstrip()
+    maxVal = f'MaxValRange={orig_val}'
+    modify_range = [(ldap.MOD_DELETE, 'lDAPAdminLimits', maxVal.encode('utf-8'))]
+    (ret, _) = ad_ldap.modify_ldap(DEF_QUERY, modify_range)
+    maxVal = 'MaxValRange=50'
+    modify_range = [(ldap.MOD_ADD, 'lDAPAdminLimits', maxVal.encode('utf-8'))]
```

Comment:
Please do this usin powershell if possible or at least on AD server machine we are avoiding direct ldap access on AD due to SSL issues.